### ISSUE TITLE
Add open folder 

### ISSUE THREAD

**Issue by NateLing** [google/xi-mac#156]
![image](https://user-images.githubusercontent.com/9606668/37759895-fb73da40-2def-11e8-877e-707ebf0dfa2f.png)

plz add folder view，This feature is useful for programmers。

**Comment by cmyr** [comment]
Yes I agree that some sort of workspace management is something we will need to consider at some point.

**Comment by jansol** [comment]
#232 addresses this at least partially.
